mock_data/remote_rooms.py

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
This file would be better if it would be a JSON file. The data we are getting from the calendar API is in JSON so it would be better to handle it that way instead of a dictionary.

Reply:
This has been done @michael-basweti